.github/workflows/inspektor-gadget.yml

```diff
@@ -822,8 +822,6 @@ jobs:
       - ebpf-objects-checks
```

Comment:
I think we should mention that we are providing the checksum file and the signature somewhere in the documentation. Maybe in a new file `docs/security.md`?

Reply:
Mmmm in another commit? I think you haven't pushed it yet 😅